src/resources/views/crud/columns/url.blade.php

```diff
@@ -0,0 +1,11 @@
+{{-- regular object attribute --}}
```

Comment:
Instead of adding an `anchor` inside the `wrapper`, we could set that wrapper element to `a` and the `href` to the link. Then the text itself (inside the anchor) can use the `limit` defined by the dev. 

Links can be huge, so for most projects I think it'll be impractical to show them in their entirety.

Then again... both of these problems above would be solved, if we transformed this column to have:
- a processing php block where they set the correct `wrapper['element']` and `wrapper['href']`
- include the `text` column 

😅 right?

Reply:
Yes. I have changed that accordingly but "disk" is causing path issue. We may need to discuss on this along with @pxpm

Reply:
Hmmm ok. Ping him on Discord to solve it please 🙏